.github/workflows/pypi-cd.yml

```diff
@@ -47,7 +47,7 @@ jobs:
                       | .RUNNER = (
                           if (.RUNNER | type == "array") 
                           then (.RUNNER | map(if . == "ephemeral" then "persistent" else . end)) 
-                          else (if . == "ephemeral" then "persistent" else . end) 
+                          else (if .RUNNER == "ephemeral" then "persistent" else .RUNNER end) 
```

Comment:
@liorsve  What issue did you faced that this change was needed?
@ikolomi , might be required in the other CD workflows as well